### ISSUE TITLE
Decide on forum for development discussions

### ISSUE THREAD

**Issue by conorpp** [SoloKeysSec/solo#2]
[Some suggestions](https://github.com/conorpp/u2f-zero/issues/76#issuecomment-420629143) that have been made:
- matrix.org
- Gitter
- Discord

I think setting up a Google group might be best for everyone, feel free to suggest otherwise though.

**Comment by 0x0ece** [comment]
Or Slack, Slack is cool :)

**Comment by mvgorcum** [comment]
In the spirit of open source I'd argue for matrix or gitter.

I like matrix a lot, but I'm clearly biased.

**Comment by MaPePeR** [comment]
Any reasons not to use GitHub?

**Comment by Zjemm** [comment]
@mvgorcum  i agree, matrix is such a wonderful open project, as is this project here, so it fits nicely.

Also matrix growing rapidly in the opensource community, and is a great platform for real time discussions. and those live discussions you dont want to get on the github issue's page.

@MaPePeR Matrix can work together with github. so you can create issue's in the github from within matrix

**Comment by nickray** [comment]
We probably won't have the resources to support this before and during the Kickstarter. I also think Matrix would fit quite nicely with our philosophy! Could even go super old school and cross link with an IRC channel :D However, I've seen the client be quite slow; perhaps this has improved.

The other issue is searchability of discussion history for newcomers to the project. A Google group might work, but that interface is quite cumbersome. For little things, I agree that Github issues are sufficient. For bigger things, perhaps a kind of "Solo Enhancement Proposal" process might make sense? With a design document sketching out a plan in advance?

We use Keybase for our internal communications, for now I suggest hopping onto https://keybase.io/team/solokeys.public if you'd like some direct communication.

**Comment by nickray** [comment]
i reserved the solokeys matrix.org group and room in case we need it. this stuf is still slooow though

**Comment by Zjemm** [comment]
Good, I'll join 😀

Let's go with matrix

**Comment by nickray** [comment]
It is only reserved, not in use :) If you wanna chat, head to https://keybase.io/team/solokeys.public for now.

**Comment by corbolais** [comment]
My feel is, keybase is not living up to FLOSS standards. W/o having dug into details, the codebase involves binary blobs. That's what's scaring me most.

Anyone fancing #solokeyssec @freenode? Waiting for *you* to /j #solokeyssec ;)

**Comment by mvgorcum** [comment]
We can link the matrix room to the freenode room, wouldn't that be the best of both worlds?

**Comment by corbolais** [comment]
dunno how matrix is operating..

**Comment by sbrl** [comment]
Keybase has binary blobs? Oh dear.

I'd prefer XMPP myself instead of IRC/freenode, but I think I might be able to set up a personal gateway there if need be.

**Comment by Krastanov** [comment]
Did a decision evolve? I am interested in a way to follow the bleeding edge development of the project, roadmaps, etc, but I have not been able to find any particularly recent forum.

**Comment by corbolais** [comment]
Hi,

Some note dislike for keybase for various reasons.

In my behalf, I try to avoid using binary blobs. Then it's counterproductive to employ proprietary SW to promote user contribution.

Please use common methods for issue, bug, feature tracking and mailing lists, IRC channels for direct exchange.

Note aside: Does anyone know whether signal's group chat feature scales with increasing user count?

C.

**Comment by stappersg** [comment]
(Breaking eight months of silence.)
Please prevent that this open issue will drain more human attention.

Decide on forum for development discussions

**Comment by sbrl** [comment]
I mean I could host an XMPP multi-user chatroom on my server, but something else would probably be more suitable.

**Comment by corbolais** [comment]
A bazillion of opensource projects use a thing called I R C. Maybe we could consider using something _like_ that, too?
